cats-backend/src/modules/mice/mice.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/sequelize';
+import { Mouse } from 'src/models/mouse.model';
+
+@Injectable()
+export class MiceService {
+  constructor(
+    @InjectModel(Mouse)
+    private mouseModel: typeof Mouse,
+  ) {}
+
+  async deleteMouse(id: number): Promise<void> {
+    await this.mouseModel.destroy({
```

Comment:
```suggestion
    return this.mouseModel.destroy({
```
 I think it's better practice to return response on delete